include/z64actor.h

```diff
@@ -511,8 +519,8 @@ typedef enum {
 #define ACTOR_FLAG_1000000       (1 << 24)
 // 
 #define ACTOR_FLAG_2000000       (1 << 25)
-// 
-#define ACTOR_FLAG_4000000       (1 << 26)
+// actor can press and hold down switches
+#define ACTOR_FLAG_CAN_HOLD_SWITCH (1 << 26)
```

Comment:
tbh, I would probably prefer press over hold. It seems more used, and for whatever reason when I see "can hold switch" I think of them physically holding a switch.

Reply:
I intuitively think of "holding" a switch as a switch being pressed only if there is an actor on top of it. Then the switch will release if the actor is no longer on it. The switch is "held" in place by the actor being on top. Sort of a "temporary set". This is subtly different than a more "permanent" press where the actor only needs to press it once and the switch remains pressed.

If I'm reading this right, then `DynaPoly_UnsetAllInteractFlags` is called right after `actor->update(actor, play);` every frame, so the switch will be released if the actor is not on it for a frame. So that's where the hold comes from.

That said, you still need to trigger the "PRESS" to begin with to apply the hold in the first place, so maybe `ACTOR_FLAG_CAN_PRESS_SWITCH` is fine too, open to more thoughts

Reply:
I like PRESS more because it matches up with things like `DynaPolyActor_IsSwitchPressed`, and yeah if an actor can press a switch, I think it is kind of implied it can hold it (since it would just repress it every frame it is on it even if it isn't able to hold it for some reason).
Though should note that it looks like OoT's https://github.com/zeldaret/oot/pull/1122 has `HOLD`. 
Also I think, but not 100% sure) `ObjOshihiki` can press switches without that flag(s) since it calls `DynaPolyActor_SetActorOnTop` manually.

Reply:
Hmm, Yeah. I'll go ahead and make the change then, but leave this thread open for discussion